### PR TITLE
chore: sign out on inactive token

### PR DESCRIPTION
**Pull request by kendallstrautman**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-kssign-out-expired-refresh-token-hashicorp.vercel.app/) 🔎
- [Asana task](url) 🎟️

## 🗒️ What

Adjusts auth behavior to sign out when we get an `inactive_token` error during the refresh flow, intead of showing the 'inactivity' modal.

### CODE REVIEW

File: src/hooks/use-authentication/index.ts [hunk 1]
```diff
@@ -69,7 +85,6 @@ const useAuthentication = (
 	// Return everything packaged up in an object
 	return {
 		error,
-		hasError,
```
Comment (kendallstrautman) [inline src/hooks/use-authentication/index.ts:72]: This wasn't used.

File: src/layouts/base-new/index.tsx [hunk 2]
```diff
@@ -89,7 +89,7 @@ const BaseNewLayout = ({
 				</div>
 			</CoreDevDotLayout>
 			<ConsentManager />
-			<AuthenticationErrorDialog />
+			{/* <AuthenticationErrorDialog /> */}
```
Comment (kendallstrautman) [inline src/layouts/base-new/index.tsx:92]:
```suggestion
			<AuthenticationErrorDialog />
```

File: src/layouts/base-new/index.tsx [hunk 3]
```diff
@@ -10,7 +10,7 @@ import createConsentManager from '@hashicorp/react-consent-manager/loader'
 import { generateTopLevelSubNavItems } from 'lib/generate-top-level-sub-nav-items'
 import useScrollPercentageAnalytics from 'hooks/use-scroll-percentage-analytics'
 import CoreDevDotLayout from 'layouts/core-dev-dot-layout'
-import AuthenticationErrorDialog from 'components/authentication-error-dialog'
+// import AuthenticationErrorDialog from 'components/authentication-error-dialog'
```
Comment (kendallstrautman) [inline src/layouts/base-new/index.tsx:13]:
```suggestion
import AuthenticationErrorDialog from 'components/authentication-error-dialog'
```

File: src/pages/api/auth/[...nextauth].ts [hunk 4]
```diff
@@ -94,13 +94,16 @@ export default NextAuth({
 					token.access_token = access_token
 					token.refresh_token = refresh_token
 				} catch (err) {
-					console.error(
-						`${AuthErrors.RefreshAccessTokenError}: failed to refresh token set`,
-						err
-					)
+					let errorType = AuthErrors.RefreshAccessTokenError
```
Comment (kendallstrautman) [inline src/pages/api/auth/[...nextauth].ts:97]: The modal still renders on the generic refresh token error.

**Review by BRKalow — APPROVED** [review]
👍 worth a try!

**Pull request merged**